using-timescaledb/continuous-aggregates.md

```diff
@@ -203,80 +252,13 @@ Each continuous aggregate uses the *bucket width* of the `time_bucket`
-is 1 hour, so if we set the `refresh_lag` to 1 hour, the
-materialization will be 2 hours behind the current time.
-
-```sql
-ALTER VIEW conditions_summary_hourly SET (
-  timescaledb.refresh_lag = '1 hour'
-);
-```
-
-Lowering the `refresh_lag` parameter means that the materialization
-follows the raw data more closely, but can cause write amplification
-that might degrade insert performance. The `refresh_lag` can also be
-set to negative values. For example, setting `refresh_lag` to
-`-bucket_width` will make the continuous aggregate materialize new
-data as soon as it is available.
-
-The `refresh_lag` parameter controls the balance between the
-on-the-fly aggregation and pre-computed aggregation when querying the
-continuous aggregate. A lower `refresh_lag` will reduce the amount of
-on-the-fly aggregation by more aggressive materialization and a higher
-`refresh_lag` will increase the amount of on-the-fly aggregation by
-not materializing as agressively.
-
->:TIP: Changing the value of `refresh_lag` is rarely necessary, particularly
-given the default behavior of [real-time aggregates](#real-time-aggregates).
-
-The `timescaledb.max_interval_per_job` determines the maximum amount
-of data processed in a single job. If there is more work to do after
-the job completes, another job will be spawned immediately. This means
-that the part of the data processed by previous jobs will be available
-even before the materialization is fully caught up.
-
-This parameter can be useful when a continuous aggregate is created on
-a hypertable that already contains data and the materialization job
-needs to catch up.
-
-The `timescaledb.ignore_invalidation_older_than` parameter controls
-how modifications (inserts, updates, and deletes) will trigger update
-of the continuous aggregate. If modifications are done to the
-hypertable, it invalidates already computed parts of the aggregate and
-the aggregate has to be updated.
-
-By default, all modifications trigger an update of the continuous
-aggregate. If the `ignore_invalidation_older_than` parameter is set to
-a duration, modifications with an older time will be ignored and not
-trigger an update of the continuous aggregate. As a result, refresh
-jobs will complete faster since they are only dealing with new data as
-part of the refresh.
-
-A common use case is to drop the raw data from the hypertable and just
-keep the downsampled data in the continuous aggregate. So, for
-example, if you want to keep all downsampled data, but drop anything
-from the raw tables that is older than 30 days, you set
-`ignore_invalidation_older_than` to `30 days` and can then drop old
-chunks from the table using `drop_chunks`:
-
-```sql
-ALTER VIEW device_readings SET (
-  timescaledb.ignore_invalidation_older_than = '30 days'
-);
-SELECT drop_chunks(INTERVAL '30 days', 'device_readings')
-```
+As a result, the materialization lags behind by one bucket and up to
```

Comment:
This seems to repeat (or be related to) the ``[end_interval, end_interval+schedule_interval)` thing above. Maybe remove/harmonize.